src/components/DateSelect.tsx

```diff
@@ -0,0 +1,38 @@
+import { Pressable, View } from "react-native";
+import Text from "./Text";
+import Calendar from "./Calendar";
+
+interface DateSelectProps {
+  date: string;
+  onDateChange: (date: string) => void;
+}
+
+export default function DateSelect({ date, onDateChange }: DateSelectProps) {
+  return (
+    <View className="bg-neutral-5 rounded-sm p-md">
+      <View className="gap-md">
+        <InboxSelect date={date} onDateChange={onDateChange} />
+        <Calendar selectedDate={date} onDateSelect={onDateChange} />
+      </View>
+    </View>
+  );
+}
+
+interface InboxSelectProps {
+  date: string;
+  onDateChange: (date: string) => void;
+}
+
+function InboxSelect({ date, onDateChange }: InboxSelectProps) {
+  const isSelected = date === "";
```

Comment:
This is a bit surprising to read. 

It isn't 100% clear to me why something should be selected if date == '';